src/components/tool/guide/Config/AlertHelper.js

```diff
@@ -5,6 +5,7 @@ export default function AlertHelper({
   alertType = "info",
   alertMsg = null,
   onClose = {},
+  children = null,
```

Comment:
I'm not really a fan of having two different props to set the same thing, as you have to know internal details of the component in order to know which prop has precedent. Is there a reason to not use one?

Reply:
I'm not convinced to have it to support having to modify the component to support child component. What's the point of having if we already have a prop that already supports message as a function or a string? Should we just keep `children` prop in here?

Reply:
Actually we can just make one props to support multiple types.

Reply:
Fixed